activemodel/lib/active_model/secure_password.rb

```diff
@@ -11,8 +13,24 @@ module SecurePassword
 
     class << self
       attr_accessor :min_cost # :nodoc:
+      attr_writer :cost # :nodoc:
+
+      def cost # :nodoc:
+        min_cost ? BCrypt::Engine::MIN_COST : @cost
+      end
```

Comment:
Note: no longer falls back to `BCrypt::Engine.cost` for global override.

We could check whether the cost was globally changed (`BCrypt::Engine.cost != BCrypt::Engine::DEFAULT_COST`) and use it if it's greater than `SecurePassword.default_cost`, but that's getting a bit too fussy.